test/ext/test_horizontal_shard.py

```diff
@@ -226,6 +226,16 @@ def test_get(self):
         t2 = sess.get(WeatherLocation, 1)
         is_(t2, tokyo)
 
+    def test_get_one(self):
```

Comment:
**mike bayer** (zzzeek) wrote:

we would need to test the NoResultFound case here also right?

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4872

Reply:
perfect, assertion on NoResultFound exception added.